File: PRO/codigos2324/SESIONES EJERCICIOS/EjerciciosExamenFinal/src/ejercicio2/model/Alumno.java
package ejercicio2.model;

public class Alumno {

    private String nombre, apellido, correo;
    private int nota;

    public Alumno() {
    }

    public Alumno(String nombre, String apellido, String correo, int nota) {
        this.nombre = nombre;
        this.apellido = apellido;
        this.correo = correo;
        this.nota = nota;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getApellido() {
        return apellido;
    }

    public void setApellido(String apellido) {
        this.apellido = apellido;
    }

    public String getCorreo() {
        return correo;
    }

    public void setCorreo(String correo) {
        this.correo = correo;
    }

    public int getNota() {
        return nota;
    }

    public void setNota(int nota) {
        this.nota = nota;
    }
}

File: FPRO/curso2526/T4/Ficheros/Alumno.cs
public class Alumno
{
    public string? nombre { get; set; }
    public string? apellido { get; set; }
    public int nota { get; set; }

    public Alumno(string nombre, string apellido, int nota)
    {
        this.nombre = nombre;
        this.apellido = apellido;
        this.nota = nota;
    }

    public Alumno()
    {

    }

    public string FormatCSV()
    {
        return string.Format("{0},{1},{2}", nombre, apellido, nota);
    }

}